Makefile

```diff
@@ -353,3 +378,35 @@ delete-minikube-profile:
 
 delete-all-minikube-profiles:
 	minikube delete --all
+
+############
+# Operator #
+############
+
+# Current Operator version
+OPERATOR_VERSION ?= 0.0.1
+
+# Generate bundle manifests and metadata, then validate generated files.
+.PHONY: operator-bundle
+operator-bundle: create-ocp-manifests
+	set -eux
+	cp ./build/assisted-installer/ocp_role.yaml config/rbac
+	cp ./build/assisted-installer/kube_api_roles.yaml config/rbac
+	cp ./build/assisted-installer/controller_roles.yaml config/rbac
+	cp ./build/assisted-installer/scality-secret.yaml config/assisted-service
+	cp ./build/assisted-installer/scality-public-secret.yaml config/assisted-service
+	cp ./build/assisted-installer/postgres-deployment.yaml config/assisted-service
+	cp ./build/assisted-installer/assisted-installer-sso.yaml config/assisted-service
+	cp ./build/assisted-installer/assisted-service-configmap.yaml config/assisted-service
+	cp ./build/assisted-installer/assisted-service-service.yaml config/assisted-service
+	cp ./build/assisted-installer/assisted-service.yaml config/assisted-service
+	cp ./build/assisted-installer/deploy_ui.yaml config/assisted-service
+	#operator-sdk generate kustomize manifests -q
+	#cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(OPERATOR_VERSION) $(BUNDLE_METADATA_OPTS)
+	operator-sdk bundle validate ./bundle
+
+# Build the bundle image.
+.PHONY: operator-bundle-build
+operator-bundle-build:
+	podman build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
```

Comment:
I don't see the `bundle.Dockerfile` anywhere. Is it generated somewhere or just missing?

Reply:
I removed it from the repo and added to .gitignore. It is generated by "make operator-build".